### ISSUE TITLE
CTD during AI turn; Invalid Reference on ArtDefine_Landmarks.LayoutHandler

### ISSUE THREAD

**Issue by broofthesun** [LoneGazebo/Community-Patch-DLL#2026]
Bug Report Template

Mod Version: 
03-09-2016

1. Mod List (if using standard CPP set, leave blank): 
Just using CPP

2. Type of error (i.e. crash, interface bug, AI quirk): 
CTD

3. Steps to reproduce:
It happens during AI's turn. Sometimes I can get around it by reloading from an autosave, and doing something different during my turn, but it will always happen again. 

4. Additional information:
Looking at the logs. The only error I see is 'Invalid Reference on ArtDefine_Landmarks.LayoutHandler - "SPECIAL" does not exist in ArtDefine_LandmarkTypes'
[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/329035/Logs.zip)

---------------------------
Supporting information:

Please note that you can attach .zip files to Issues by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files: Database.log and Lua.log needed
2. Minidump file (located in your Civ5.exe directory)
3. Screenshots (if needed)

**Comment by LoneGazebo** [comment]
Those database errors are benign (standard civ has them too).

**Comment by broofthesun** [comment]
Thanks for the quick response. Any idea what could be causing the CTD? One other thing I noticed is that it seems to happen when an AI civ wants to say something to me (the player), and the game goes to the screen showing the AI civ.

Another random idea on what may be causing this, I installed one of the CPP betas after 3-9. This had some bugs, and was causing CTDs. So I tried doing a clean install of CPP back to the 3-9 version (following the instructions in Manual created by Estebanium). Before installing the beta, I did not notice any CTDs. Anything here that may be causing the CTDs? I can't say for sure that this has any effect on causing them.

I have tried clean installs of CPP, and using Steam to verify Civ 5 local game files. The only other thing I have to try is completely uninstalling, then re-installing Civ 5 and CPP. However, my internet connection is slow, and I wanted to post this issue first before going that route.

**Comment by Edaka** [comment]
I'd recommend reinstalling - somebody has reportedly had success with that method. Also, correct me if I'm wrong, but your attached archive seems to be empty.

**Comment by broofthesun** [comment]
Hi, thank you for the input. I tried a full reinstall of Civ 5 and CPP, and that did not resolve the issue (CTD on turn 2 of a new game). 

However, I think I did finally fix the issue. I believe it was related to playing the game in windowed mode. I switched from playing in windowed mode to fullscreen, and no longer have CTD issues. I was able to play a full game without any CTDs. Not sure why playing in windowed mode would cause the issue. I can't say for sure that this is the exact cause without more testing, but it looks resolved so far.
